### PR TITLE
Add a guaranteed cotton pizza to pizza crates

### PR DESCRIPTION
**Pull request by ewokswagger**
## About the PR
Replace one of the random pizzas from emergency/disaster pizza deliveries with a cotton one.

## Why / Balance
Guarantees that all species get something edible from pizza deliveries.

## Technical details
Added a new prototype for cotton pizza boxes and replaced one of the random pizzas in the crates with the cotton one.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Pizza deliveries are now guaranteed to have at least one cotton pizza.

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Crates/food.yml [hunk 1]
```diff
@@ -20,7 +21,8 @@
   - type: StorageFill
     contents:
     - id: FoodBoxPizzaFilled
-      amount: 16
+      amount: 15
+    - id: FoodBoxPizzaCotton
```
Comment (PJB3005) [inline Resources/Prototypes/Catalog/Fills/Crates/food.yml:25]: Wouldn't it make sense to have like, two or three of these in the large crate?

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml [hunk 2]
```diff
@@ -274,6 +274,17 @@
       prob: 0.10
       orGroup: Pizza
     - id: KnifePlastic
+  
+- type: entity
+  name: pizza box
+  parent: FoodBoxPizzaFilled
+  id: FoodBoxPizzaCotton
+  suffix: Filled
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml:282]: Change this so it's clearly shown as a cotton pizza in the spawn menu.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Crates/food.yml (hunk 1 above)
Comment (ewokswagger) [reply Resources/Prototypes/Catalog/Fills/Crates/food.yml:25]: It is something I considered. Right now we have about 2/16 on average with this change which feels like a fair perportion given its about 1/8 species. If anything, the small pizza box overrepesents cotton pizzas, but I decided to prioritize there is at least something for everyone in this case. I am happy to change it though if you feel 2-3 feels more appropriate.

Comment (PJB3005) [reply Resources/Prototypes/Catalog/Fills/Crates/food.yml:25]: Oh right the thing can already spawn one, right.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml (hunk 2 above)
Comment (ewokswagger) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml]: This change has been pushed

File: Resources/Prototypes/Catalog/Fills/Crates/food.yml (hunk 1 above)
Comment (ewokswagger) [reply Resources/Prototypes/Catalog/Fills/Crates/food.yml:25]: I can add a comment to clarify the reasoning if needed

Comment (PJB3005) [reply Resources/Prototypes/Catalog/Fills/Crates/food.yml:25]: Nah you're good.

**Review by PJB3005 — APPROVED** [review]

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**